site/content/docs/latest/README.md

```diff
@@ -0,0 +1,16 @@
+# Kubeapps documentation
+
+Everything you need to know about Kubeapps.
+
+> NOTE: we are currently moving our docs to a new website and reviewing our documentation files. In the meanwhile, eventual  broken links and minor inconsisteces may appear. Please, feel free to contact us if you have any questions: file an [issue](https://github.com/vmware-tanzu/kubeapps/issues), or talk to us on the [#Kubeapps slack channel](https://kubernetes.slack.com/messages/kubeapps).
```

Comment:
Plenty of the links to md files or yaml/js files won't work. However, being a website, it is really easy to use a 3rd party service to analyze dead links. For instance: https://www.drlinkcheck.com/